src/icetransport.cpp

```diff
@@ -171,7 +171,10 @@ bool IceTransport::send(message_ptr message) {
 
 void IceTransport::incoming(message_ptr message) {
 	PLOG_VERBOSE << "Incoming size=" << message->size();
-	recv(message);
+	if (mRecvCallback)
```

Comment:
`mRecvCallback` is private in the parent class `Transport`, you shouldn't be able to access it here.

Reply:
Yes pasted to wrong place :)